src/jit/codegencommon.cpp

```diff
@@ -8644,6 +8644,43 @@ void CodeGen::genFnEpilog(BasicBlock* block)
         unwindStarted = true;
     }
 
+#if !FEATURE_FASTTAILCALL
+    if (jmpEpilog)
+    {
+        GenTree* jmpNode = block->lastNode();
+
+        noway_assert(jmpNode->gtOper == GT_JMP);
+
+        CORINFO_METHOD_HANDLE methHnd = (CORINFO_METHOD_HANDLE)jmpNode->gtVal.gtVal1;
+        CORINFO_CONST_LOOKUP  addrInfo;
+        compiler->info.compCompHnd->getFunctionEntryPoint(methHnd, &addrInfo);
+
+        if (addrInfo.accessType == IAT_RELPVALUE)
+        {
+            // IAT_RELPVALUE jump at the end is done using relative indirection, so,
+            // additional helper register is required.
+            // We use LR just before it is going to be restored from stack, i.e.
+            //
+            //     movw r12, laddr
```

Comment:
This is very ARM32-specific. Should this block rather be under ARM32 ifdef? It happens to work today because of `FEATURE_FASTTAILCALL` is not defined for ARM32. If somebody undefines `FEATURE_FASTTAILCALL` for ARM64 e.g. for testing, things will break.